packer/install_client.sh

```diff
@@ -17,8 +17,10 @@ sed -i "s/passkey v=''/passkey v='$fah_passkey'/; \
      s/user v=''/user v='$fah_user'/" /tmp/config.xml;
 
 sudo service FAHClient stop;
+sleep 10;
```

Comment:
Local tests/runs showed that the config file was still not being replaced correctly :(. Adding a sleep before/after helped, but I'm still not confident that this will fully work.

Reply:
bummer ;/